prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -299,6 +323,50 @@ func popRandom(set sets.String) string {
 	return sel
 }
 
+// findNonBusyReviewer finds a non-busy reviewer from a set.
+func findNonBusyReviewer(ghc githubClient, log *logrus.Entry, busyReviewers sets.String, set sets.String) string {
```

Comment:
Would busyReviewers *sets.String make more sense here?

Otherwise if busyReviewers, set are both the same type you only need to specify it once